examples/todo-intermediate/src/application.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/example-todo
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {ApplicationConfig, Provider, Constructor} from '@loopback/core';
+import {RestApplication} from '@loopback/rest';
+import {MySequence} from './sequence';
+import {GeocoderServiceProvider} from './services';
+
+/* tslint:disable:no-unused-variable */
+// Binding and Booter imports are required to infer types for BootMixin!
+import {BootMixin, Booter, Binding} from '@loopback/boot';
+
+// juggler imports are required to infer types for RepositoryMixin!
+import {
+  Class,
+  Repository,
+  RepositoryMixin,
+  juggler,
+} from '@loopback/repository';
+/* tslint:enable:no-unused-variable */
+
+export class TodoListApplication extends BootMixin(
+  RepositoryMixin(RestApplication),
+) {
+  constructor(options?: ApplicationConfig) {
+    super(options);
+
+    // Set up the custom sequence
+    this.sequence(MySequence);
+
+    this.projectRoot = __dirname;
+    // Customize @loopback/boot Booter Conventions here
+    this.bootOptions = {
+      controllers: {
+        // Customize ControllerBooter Conventions here
+        dirs: ['controllers'],
+        extensions: ['.controller.js'],
+        nested: true,
+      },
+    };
+
+    // TODO(bajtos) Services should be created and registered by @loopback/boot
+    // See https://github.com/strongloop/loopback-next/issues/1439
+    this.setupServices();
+  }
+
+  setupServices() {
+    this.service(GeocoderServiceProvider);
```

Comment:
Since the primary goal of `todo-intermediate` is to show model relations, I am proposing to simplify the `Todo` model and remove geo lookup:

 1. There will be less moving parts to maintain.
 2. Smaller codebase will make it easier for LB4 user to understand what are the relevant bits to learn from this example
 3. The geocoder service slows down the test suite, we don't want to have twice as many slow tests as we already have.